btcec/signature.go

```diff
@@ -352,6 +352,10 @@ func SignCompact(curve *KoblitzCurve, key *PrivateKey,
 	// bitcoind checks the bit length of R and S here. The ecdsa signature
 	// algorithm returns R and S mod N therefore they will be the bitsize of
 	// the curve, and thus correctly sized.
+	return makeCompact(curve, sig, key, hash, isCompressedKey)
+}
+
+func makeCompact(curve *KoblitzCurve, sig *Signature, key *PrivateKey, hash []byte, isCompressedKey bool) ([]byte, error) {
```

Comment:
There is no documentation on this one.

And the method is used before it is declared.